hack/testing/logging.sh

```diff
@@ -261,8 +261,11 @@ if [ "$ENABLE_OPS_CLUSTER" = "true" ] ; then
          system:serviceaccount:logging:aggregated-logging-elasticsearch"
     # update the ES_OPS DC to be in the privileged context
     # TODO: should not have to do this - should work the same as regular hostmount
-    os::cmd::expect_success "oc patch $(oc get dc -o name -l component=es-ops) \
+    ops_dc=$(oc get dc -o name -l component=es-ops)
+    os::cmd::expect_success "oc patch $ops_dc \
        -p '{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"elasticsearch\",\"securityContext\":{\"privileged\": true}}]}}}}'"
+
+    os::cmd::try_until_text "oc deploy $ops_dc --latest" "Started" "$(( 3 * TIME_MIN ))"
```

Comment:
I guess since this is logging.sh there's only gonna be one, so it doesn't matter.